hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakeInfoHelper.java

```diff
@@ -190,13 +181,13 @@ public void withdrawStake(
      * Also clears any pending rewards from the {@link NetworkStakingRewards} singleton for nodes that are no
      * longer in the address book.
      *
-     * @param context      the token context
-     * @param networkInfo  the list of node infos from the address book
-     * @param config       the configuration for the node
-     * @param infoStore    the writable store for the staking info
+     * @param context the token context
+     * @param networkInfo the list of node infos from the address book
+     * @param config the configuration for the node
+     * @param infoStore the writable store for the staking info
      * @param rewardsStore the store for the staking rewards
      */
-    public StreamBuilder adjustPostUpgradeStakes(
+    public void adjustPostUpgradeStakes(
```

Comment:
Can we remove this? It seems like we added this in this PR https://github.com/hashgraph/hedera-services/pull/15444 . Trying to remember why we added it.